### DISCUSSION TITLE
Is it possible to port to VSCode as well?

### DISCUSSION THREAD

**Discussion by PrashanthShetty** [rmehri01/onenord.nvim#72] [General]
Hello,
thank you so much for the theme, I am enjoying it more than any another themes. I used it with warp term, vim  and now looking for VSCode as well. please let me know how to port or use it.

Regards,
P.

**Comment by rmehri01** [comment]
Hey again! Someone actually already did this and it's available on the VSCode marketplace: https://github.com/s1e2b3i4/onenord-vscode.

**Reply by PrashanthShetty** [reply]
Thank you, I thought it is available in the extension marketplace. So I have to download the json and set it?

**Reply by rmehri01** [reply]
I don't think you need to download the json, if you just download it from the marketplace it should show up automatically: https://marketplace.visualstudio.com/items?itemName=s1e2b3i4.onenord-vscode
